lib/tests/test_revset.rs

```diff
@@ -12,11 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// TODO: Remove after https://github.com/frondeus/test-case/issues/122 is resolved
```

Comment:
Re commit message: To be clear, only the `allow(unknown_lints)` part was there for clippy < 1.70. The actual problem affected nighlites and `cargo +beta clippy` until a week ago.

But I think https://github.com/frondeus/test-case/issues/122 is fixed now.